src/com/mambu/apisdk/services/LinesOfCreditService.java

```diff
@@ -158,7 +180,7 @@ public LineOfCredit getLineOfCreditDetails(String lineofcreditId) throws MambuAp
 		switch (customerType) {
 		case CLIENT:
 		case GROUP:
-			return serviceExecutor.getOwnedEntities(customerType, customerId, serviceEntity, offset, limit);
+			return serviceExecutor.getOwnedEntities(customerType, customerId, serviceEntity, offset, limit, requresFullDetails);
```

Comment:
This method has 6 parameters. In the future, maybe we can consider grouping "offset" and "limit" in a single object, like Pagination. How does that sound to you? :)